o2a/converter/oozie_converter.py

```diff
@@ -68,33 +67,28 @@ def __init__(
         self.input_directory_path = input_directory_path
         self.output_directory_path = output_directory_path
         self.dag_name = dag_name
-        self.config_file = os.path.join(input_directory_path, CONFIG)
-        self.job_properties_file = os.path.join(input_directory_path, JOB_PROPS)
         self.renderer = renderer
         self.transformers = transformers or []
         # Propagate the configuration in case initial property set is passed
-        self.job_properties = {} if not initial_props else initial_props.job_properties
-        self.job_properties["user.name"] = user or os.environ["USER"]
-        self.config: Dict[str, str] = {}
+        job_properties = {} if not initial_props else initial_props.job_properties
+        job_properties["user.name"] = user or os.environ["USER"]
+        self.props = PropertySet(job_properties=job_properties)
         self.workflow = Workflow(
             dag_name=dag_name,
             input_directory_path=input_directory_path,
             output_directory_path=output_directory_path,
         )
-        self.props = PropertySet(
-            job_properties=self.job_properties, config=self.config, action_node_properties={}
-        )
-        self.read_and_update_job_properties_replace_el()
-        self.read_config_replace_el()
         self.parser = parser.OozieParser(
             props=self.props, action_mapper=action_mapper, renderer=self.renderer, workflow=self.workflow
         )
+        self.property_parser = PropertyParser(props=self.props, workflow=self.workflow)
 
     def recreate_output_directory(self):
         shutil.rmtree(self.output_directory_path, ignore_errors=True)
         os.makedirs(self.output_directory_path, exist_ok=True)
 
     def convert(self, as_subworkflow=False):
+        self.property_parser.parse_property()
```

Comment:
This is important because it organizes the conversation process. Now in the documentation it will be easier to describe what happens with the properties files.

This is safe because in the OozieParser constructor this object is not modified.